db/migrations/0001_initial.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.0.2 on 2023-10-16 11:52
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Rase',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
change ' to ", use Black

Reply:
migrations were generated automaticly and Arsen told then i can leave this